src/DevSoc.Eventathon.Users/IUsersRepository.cs

```diff
@@ -5,7 +5,7 @@ namespace DevSoc.Eventathon.Data;
 
 public interface IUsersRepository
 {
-    Task<User?> GetUser(string id);
+    Task<List<User>> GetUser(string id);
```

Comment:
Why would you return a list of users for a single ID?

Reply:
Because I tried it without and couldn't get it working, I was gonna ask you to have a look at it tbh